### ISSUE TITLE
Allow multiple instances to be started during deploy

### ISSUE THREAD

**Issue by bmorton** [bmorton/deployster#6]
Fleet supports starting lots of units via a [template](https://coreos.com/docs/launching-containers/launching/fleet-unit-files/#template-unit-files) (`name@.service` -> `name@{1..3}.service`).  Currently, this deploy service only starts up `name-version@1.service`.  We should support some configurable amount of instances, possibly contained in the deploy payload.

Other ideas and considerations:

* It'd be weird to have to know the number of instances you wanted to fire up every time.  We could default to replacing the currently running number of instances with the same number of the new version.
* When we're waiting for instances to come online before destroying the old instance, should we wait for all to finish or kill one-by-one as they come back?  Maybe multiple strategies?

**Comment by bmorton** [comment]
I'm making progress on this.  I did a quick spike with a very rough implementation and ended up with lots of units failing to start.  I regrouped and started digging through `fleet` and `fleetctl`'s source to make sure that I was implementing things the same way.  I wasn't, so I decided to make my implementation look more like `fleetctl` and on the way decided to ditch the fleet package I had created in deployster.  This is how we ended up with #29.

After having done all that work, it still fails in the same way I was seeing previously and I have been able to reproduce this using `fleetctl` as well.  I've opened a ticket with coreos/fleet#1139 to track this.

In the meantime, I'm experimenting with changing the unit naming convention to include a timestamp like I mentioned in the issue above.  It started out as a workaround, but I might actually like having it there anyway.  I'm undecided still, but this is letting me move forward a bit.

Old naming convention:
`railsapp-1bbc321@1.service`

New (experimental, proposed) naming convention:
`railsapp:1bbc321:2015-02-25T00.50.01Z@1.service`

I'm in the middle of exploring how to make that new naming convention work.
